sensing/pointcloud_preprocessor/docs/ring-outlier-filter.md

```diff
@@ -22,14 +22,14 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 
 ### Core Parameters
 
-| Name                      | Type    | Default Value | Description                                                                                                                     |
-| ------------------------- | ------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------- |
-| `distance_ratio`          | double  | 1.03          |                                                                                                                                 |
-| `object_length_threshold` | double  | 0.1           |                                                                                                                                 |
-| `num_points_threshold`    | int     | 4             |                                                                                                                                 |
-| `max_rings_num`           | uint_16 | 128           |                                                                                                                                 |
-| `max_points_num_per_ring` | size_t  | 4000          | Set this value large enough such that `HFoV / resolution < max_points_num_per_ring`                                             |
-| `publish_excluded_points` | bool    | false         | Flag to publish excluded pointcloud for debugging purpose. Due to performance concerns, please set to false during experiments. |
+| Name                      | Type    | Default Value | Description                                                                                             |
+| ------------------------- | ------- | ------------- | ------------------------------------------------------------------------------------------------------- |
+| `distance_ratio`          | double  | 1.03          |                                                                                                         |
+| `object_length_threshold` | double  | 0.1           |                                                                                                         |
+| `num_points_threshold`    | int     | 4             |                                                                                                         |
+| `max_rings_num`           | uint_16 | 128           |                                                                                                         |
+| `max_points_num_per_ring` | size_t  | 4000          | Set this value large enough such that `HFoV / resolution < max_points_num_per_ring`                     |
+| `publish_noise_points`    | bool    | false         | Flag to publish point cloud noise. Due to performance concerns, please set to false during experiments. |
```

Comment:
Since the node name is `ring outlier filter`, it is better to unify the name `outlier`.

Reply:
Yes :+1:

Reply:
Thanks! And agree with your opinion!!
I will change them.

Reply:
```suggestion
| `publish_outlier_pointcloud`    | bool    | false         | Flag to publish point cloud noise. Due to performance concerns, please set to false during experiments. |
```

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/6581/commits/d85eabb2b687547659cf6763787cf936bd2b5592